src/main/minio.js

```diff
@@ -1194,29 +1194,47 @@ export class Client {
     if (!isString(marker)) {
       throw new TypeError('marker should be of type "string"')
```

Comment:
@prakash,
Run your test with `MaxKeys: 1` and no recursive listing, and type error (pointed line, #1195) will be thrown.
I think you need to have more than one object and a prefix in your `<bucket>/<prefixes>]` location where you do your listing.

```
/home/ersan/work/src/github.com/minio/minio-js/dist/main/minio.js:1461
        throw new TypeError('marker should be of type "string"');
        ^

TypeError: marker should be of type "string"
    at Client.listObjectsQuery (/home/ersan/work/src/github.com/minio/minio-js/dist/main/minio.js:1461:15)
    at Readable.readStream._read (/home/ersan/work/src/github.com/minio/minio-js/dist/main/minio.js:1596:16)
    at Readable.read (_stream_readable.js:475:10)
    at maybeReadMore_ (_stream_readable.js:623:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

Reply:
I have updated the piece of code. PTAL